src/pages/index.js

```diff
@@ -1,275 +1,242 @@
-    {name: "7 Wonders", link: "https://vigilantebar.com/wp-content/uploads/2020/06/pic860217.jpg"},
-    {name: "Abduction", link: "https://vigilantebar.com/wp-content/uploads/2023/08/71Z8QTM8qXL._AC_UF8941000_QL80_.jpg"},
-    {name: "Agricola", link: "https://vigilantebar.com/wp-content/uploads/2022/08/Agricola.jpg"},
-    {name: "Black Fleet", link: "https://vigilantebar.com/wp-content/uploads/2022/08/Black-Fleet.jpg"},
-    {name: "Call to Adventure", link: "https://vigilantebar.com/wp-content/uploads/2022/08/Call-to-Adventure-scaled.jpg"}
-]
-
-const cardsList = document.querySelector('.cards__list');
-initialCards.forEach(cardData => {
-    cardsList.append(getCardElement(cardData));
+import ModalWithForm from "../components/ModalWithForm.js";
+import ModalPreviewImage from "../components/ModalPreviewImage.js";
+import Section from "../components/Section.js";
+import FormValidator from "../scripts/validation.js";
+import Api from "../components/Api.js";
+import UserProfile from "../components/UserProfile.js";
+import ModalConfirm from "../components/ModalConfirm.js";
+import { v4 as uuidv4 } from "uuid";
+
+const dbApi = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "0ba3486c-b117-4196-b741-be8eda3e197d",
+    "Content-Type": "application/json",
+  },
 });
 
-//#endregion
-
-//#region profile modal declarations
-const profileModal = document.querySelector('#edit-profile-modal');
-const profileName = profileModal.querySelector('#profile-name');
-const profileDescription = profileModal.querySelector('#profile-description');
-profileModal.addEventListener('click',checkForModalOverlayClick);
-document.querySelector('.profile__edit-button').addEventListener('click',onEditProfileClick);
-document.querySelector('#profile-modal__close').addEventListener('click',onCloseModalClick);
-document.forms['profile-modal__form'].addEventListener('submit',onProfileModalSubmit);
-const currentProfileName = document.querySelector('.profile__name');
-const currentProfileDescription = document.querySelector('.profile__description');
-//#endregion
+const cardSection = new Section({
+  containerSelector: `.cards__list`,
+  createItem: (data) => {
+    return makeNewCard(data);
+  },
+  initialize: (route) => {
+    dbApi.getBatchData(route).then((data) => {
+      cardSection.createItems(data);
+      cardSection.renderItems();
+    });
+  },
+});
 
-//#region new post modal declarations
-const defaultLinkField = 'Paste a link to the picture';
-const defaultCaptionField = 'Type your caption'
+cardSection.initialize(routeData.getCards);
+
+const userProfile = new UserProfile({
+  initialize: () => {
+    dbApi
+      .request(routeData.getUserInfo)
+      .then((data) => {
+        userProfile.setProfileData({
+          userName: data.name,
+          profileDescription: data.about,
+          imageUrl: data.avatar,
+        });
+        editProfileForm.setDefaultInputs(userProfile.getUserProfile());
```

Comment:
`        editProfileForm.setDefaultInputs(userProfile.getUserProfile());
`

It should be on every opening 

* See here https://snipboard.io/OhCqu2.jpg You need to **insert** the profile data into the inputs when you open the **profile popup** to let the user **edit** the data and submit it. But after the 1st submition there is no data on opening

Reply:
Fixed!